server/tests/backends/mongo_translator/test_mongo_translator_steps.py

```diff
@@ -44,12 +46,27 @@ def mongo_database(mongo_server_port):
     return client['tests']
 
 
+def _sanitize_nulls(data: List[dict], inplace: bool = True) -> List[dict]:
+    """Replaces NaTs and NaNs with None"""
+    d = data if inplace else deepcopy(data)
+    for record in d:
+        for k, v in record.items():
+            # cannot use isinstance(v, pd.NaT) here, as pd.NaT is not
+            # a type
+            if v is pd.NaT:
```

Comment:
And for this : 
```
isinstance(fg, type(pd.NaT))
```


PS : not tested !

Reply:
You're right on this. This implementation has problems anyway, so I'll just drop the commit! And since this is needed elsewhere, I created a separate ticket and proposed an implementation here: https://github.com/ToucanToco/weaverbird/pull/1316